Created by blog.html

rybakma1 opened the pull request.

rybakma1 committed:
Created by blog.html

Сверстал страницу blog.html

rybakma1 committed:
changed some classes

сделал один класс для одинаковых элемнтов на разных страницах

rybakma1 committed:
Created by toys.html

Сверстал страницу toys.html

rybakma1 committed:
Add basic styles

Две картинки в макете в png формате имеют белый фон. На мой взгляд сами картинки необходимо почистить от фона.

FedchenkoVyacheslav commented on blog.html:
```diff
@@ -0,0 +1,156 @@
+    <main>
+    <h1 class="page-title"> Blog</h1>
+    <div class="breadcrumbs-block">
+        <p class="home-page-inactive">Home</p>
```
<p class="home-page-inactive"> тут странный класс. не вписывается в контекст блока

rybakma1 replied:
Исправил

FedchenkoVyacheslav commented on blog.html:
```diff
@@ -0,0 +1,156 @@
+    <h1 class="page-title"> Blog</h1>
+    <div class="breadcrumbs-block">
+        <p class="home-page-inactive">Home</p>
+        <img src="/images/breadcrumbs-vector.png" alt="breadcrumbs vector" width="8" height="12">
```
это можно добавить как псевдоэлемент и убрать из верстки

rybakma1 replied:
Исправил.

FedchenkoVyacheslav commented on blog.html:
```diff
@@ -0,0 +1,156 @@
+    <div class="breadcrumbs-block">
+        <p class="home-page-inactive">Home</p>
+        <img src="/images/breadcrumbs-vector.png" alt="breadcrumbs vector" width="8" height="12">
+        <p class="active-page"> Blog</p>
```
Если блок "breadcrumbs-block" то пусть <p class="breadcrumbs-active">, и блок тоже можно просто breadcrumbs назвать

rybakma1 replied:
Исправил

FedchenkoVyacheslav commented on blog.html:
```diff
@@ -0,0 +1,156 @@
+    <section>
+        <h2 class="visually-hidden">Page pagination</h2>
+        <div class="left-pagination-arrow">
+            <a class="left-pagination-control" href="#"><img src="/images/left-arrow.png" alt="left arrow"></a>
```
кнопки вперед назад будут button а не ссылка

rybakma1 replied:
Исправил

rybakma1 replied:
Исправил

FedchenkoVyacheslav commented on blog.html:
```diff
@@ -0,0 +1,156 @@
+            </li>
+            <li class="blog-pagination-number">
+                <a class="blog-number-inactive blog-number" href="#">4</a>
+            </li>
```
тут тоже не тег <a> у элементов а <button>

rybakma1 replied:
Исправил

FedchenkoVyacheslav commented on toys.html:
```diff
@@ -0,0 +1,222 @@
+                <div class="cost-value">
+                    <p>0</p>
+                    <p>500</p>
+                </div>
```
тут должен быть <input type="range"> почитай об этом

rybakma1 replied:
Исправил на input (type: range;)

FedchenkoVyacheslav commented on toys.html:
```diff
@@ -0,0 +1,222 @@
+            <div class="cost-goods">
+                <div class="cost-title">
+                    <p>Cost</p>
+                </div>
```
это должен быть тег <legend>

rybakma1 replied:
Исправил

FedchenkoVyacheslav commented on toys.html:
```diff
@@ -0,0 +1,222 @@
+                <img class="goods-img" src="/images/chair-cover.png" alt="chair cover" width="380" height="385">
+                <p class="goods-description">Snarky bean bag chair cover</p>
+                <p class="goods-cost">$60.00 USD</p>
+            </li>
```
все эти карточки это ссылки на товар. при клике должен совершаться переход на страницу товара

rybakma1 replied:
Исправил: добавил ссылки к картинкам товара

FedchenkoVyacheslav commented on toys.html:
```diff
@@ -0,0 +1,222 @@
+        <div class="right-pagination-arrow">
+            <a class="right-pagination-control" href="#"><img src="/images/right-arrow.png" alt="right arrow"></a>
+        </div>
+    </section>
```
про тег <button> написал в странице blog

rybakma1 replied:
Исправил

FedchenkoVyacheslav commented on toys.html:
```diff
@@ -0,0 +1,222 @@
+        <p class="home-page-inactive">Catalog</p>
+        <img src="/images/breadcrumbs-vector.png" alt="breadcrumbs vector" width="8" height="12">
+        <p class="active-page">Toys</p>
+    </div>
```
хлебные крошки поправь как в blog

rybakma1 replied:
Исправил

rybakma1 committed:
Create grid on the undex.html

Стилизовал сетку для крупных блоков.

rybakma1 committed:
Full stiles to index.html

За счет z-index один div удалил.
Из недоделок по странице следующее:
1. В звездном рейтинге не понял еще как отступы между звездами сделать;
2. У кнопки формы отправки Email не знаю как опустить ниже первую букву;
3. В ссылку на инстаграмм не знаю как картинку сделать - в макете скачиваю svg, а они пустые;
4. Саму форму под футером еще не стилизовал и скрипт не подключал.

rybakma1 committed:
Full stiles to blog.html and catalog.html

На главной:
1. Еще осталось убрать у слайдера одну обертку - пока руки не дошли.
2. Иконку поиска в хедере сделал баттоном и реагирующей на курсор.
3. Заголовки у блока about us исправил отрицательным margin top.
4. В хедере иконки социальных сетей доделал.
5. В хедере ссылку на разработчика сайта добавил.
6. у кнопки отправки емейла шрифт добавил.
7. Попробовал background-size и width поменять в стилях звездного рейтинга, но так и не догадался какие именно нужно менять  - нужно подсказывать.
8. Страницы blog.html and catalog.html кажется полностью сделал на этом этапе - останется эффекты при наведении и клике внедрить.

rybakma1 committed:
edded popup form stile and js

С ява скриптом сдела все, как Егор показывал, но не добился работы следующих функций:
1. При открытии формы первый инпут не смог сделать в фокусе.
2. При открытой форме не смог сделат закрытие по клику на фон вокруг формы.

Возможно из-за того, что пришлось использовать для обертки формы z-index: 1; Без него картинка слайдера вставала поверх формы и ее фона.

rybakma1 committed:
Create contact-us.html and fixed curator's comments

rybakma1 committed:
Added input (type: range) to toys.html

Пока еще не смог добавить изменение стиля полосы при прогрессе бегунка на инпуте.

rybakma1 committed:
Added style to toys.html

На радиокнопке не смог вывести лейблы поверх добавленных стилей  (0-5; 10-20; 20-50)

rybakma1 committed:
Fixed some mistakes

Ховеры в данный момент доделываю.

rybakma1 committed:
Fixed bugs from the curator's comments

Осталось некоторые ховеры навести, потом всем active и click.
И потом по pixel perfect выравнивать.
И еще надо для фокуса стили, но в стайлгайде их нет. Их как в учебном проекте сделать?

rybakma1 committed:
Added "hover", "active" and "focus" effects

У форм цвет рамки красный и желтый, чтобы визуально заметно было работу фокуса. Потом заменю на темно серый, как у инпутов (но его почти не заметно).

rybakma1 committed:
Added all effects

При добавлении еффекта на "Company rating" пришлось добавить дополнительный div после инпутов. Но при этом legend пришлось скрыть. Теперь в футере при клике по слову "Company Rating" не встает фокус на инпуты, т.к. это уже не Легенд, а псевдоэлемент.

rybakma1 committed:
Added min.css

Заметил, что при клике по кнопке в слайдере форма открывается с фокусом на первом инпуте, но тут же закрывается сама.
Похоже я ссылку на баттон неверно поменял.

rybakma1 committed:
Correction of writing fonts

rybakma1 committed:
Final fixes

rybakma1 committed:
Fix validation errors

Пофиксил ошибки валидации и  подключил недостающий шрифт.

rybakma1 committed:
Fixed text movement on hover
